### DISCUSSION TITLE
singing manual

### DISCUSSION THREAD

**Discussion by king-dahmanus** [dectalk/dectalk#13] [Ideas]
Hey, thanks  for reserecting this tts engine. Will you make some manuals to teach  us how to manipulate the parameters like pich, inflexion, vibrato and phonemes in order to achieve unconventional results like singing?

**Comment by 7coil** [comment]
There used to be a website two years ago that was part of this organisation that had a collection of pre-made songs in dectalk/list - It might be nice to resurrect it and theoretically put a tutorial there.

**Reply by lawrenceper** [reply]
Hello all,

I'm knowledgeable about how to make DecTalk sing, having written some phoneme data myself. For me, I learned through trial and error, and by examining some .en files to better understand the correct syntax, and reading a tutorial from a university that explained how the different phonemes, durations, and pitches worked. Not sure if that site is still up today.

I noticed that in the /dist/doc directory after compilation, there appear to be some documentation in various formats like HTML and PDF. this appears to be the DECtalk manual. Within that documentation, I saw a reference guide for DecTalk commands, as well as a complete table of phonemes, although slightly confusing to read with a screen reader. I recall the university website having a similar list, from which I would type in the phonemes at random into DecTalk to understand what sound it made, but that was many years ago. If you'd like an audio tutorial, please let me know.

Hope this helps.

Regards,
Lawrence
